lib/r2d2/util.rb

```diff
@@ -10,7 +10,7 @@ def build_token(token_attrs, recipient_id: nil, verification_keys: nil)
     case protocol_version
```

Comment:
I believe the author originally put this in `Util` because it it's also used for the `AndroidPayToken` class. That workflow has been deprecated by Google, so we're thinking about going in and ripping that functionality out. At that point, it would probably make a lot of sense to move methods out of `Util`.